05_syntax_lab/01.py

```diff
@@ -1,19 +1,5 @@
-"""
-Write a program that reads 10 numbers from
-the user and prints the largest one
-
-For example if the largest number a user has typed
-is 74, program should print:
-
-    Max number = 74
-"""
-
-maxnum = float('-inf')
-
-for _ in range(10):
-    num = float(raw_input())
-    if num > maxnum:
-        maxnum = num
-
-print "Max number = %f" % maxnum
-
+#!/usr/bin/python
+"""This script takes 10 numbers from user returens the highest one"""
+listOfNumbers = []
+for num in range(10): listOfNumbers.append(float(raw_input('input number: ')))	
+print 'max number is: %f' % max(listOfNumbers) 
```

Comment:
Works well, though since we didn't learn lists yet I had a different solution in mind - using a loop to test if each number is larger than the previous one

One note - in python we usually prefer the snake case versio: `list_of_numbers` over the camel case `listOfNumbers` for variable and function names